react/UploadQueue/Readme.md

```diff
@@ -18,13 +33,16 @@ const state = {
   successCount: 1
 };
 
-<UploadQueue
-  lang='fr'
-  app='Cozy Drive'
-  getMimeTypeIcon={() => 'file'}
-  queue={state.queue}
-  doneCount={state.doneCount}
-  successCount={state.successCount}
-  popover={false}
-/>
+<>
+  popover: <input type="checkbox" value={state.popover} onChange={() => setState({ popover: !state.popover })} />
+  <UploadQueue
+    lang='fr'
+    app='Cozy Drive'
+    getMimeTypeIcon={() => 'file'}
+    queue={data.queue}
+    doneCount={data.doneCount}
+    successCount={data.successCount}
+    popover={state.popover}
+  />
+</>
```

Comment:
I'm not sure that having an interactive exemple is a good idea for our Argos test. If I'm not mistaken we're not screenshoting when popover={true}, right?

Reply:
> I'm not sure that having an interactive exemple is a good idea for our Argos test

Why ?

The checkbox is for the developer working on the UploadQueue.

Reply:
Because we are not screenshoting the component when popover={true} and I think it's not a good idea to no doing it.

Reply:
Since this is not the main point of this PR, I prefer to improve the screenshots in another PR https://github.com/cozy/cozy-ui/pull/1416, this way I can merge this fix earlier.